src/api/java/com/minecolonies/api/client/render/modeltype/ModModelTypes.java

```diff
@@ -0,0 +1,84 @@
+    public static final ResourceLocation CHILD_ID          = new ResourceLocation(Constants.MOD_ID, "child");
+    public static final ResourceLocation HEALER_ID         = new ResourceLocation(Constants.MOD_ID, "healer");
+    public static final ResourceLocation TEACHER_ID        = new ResourceLocation(Constants.MOD_ID, "teacher");
+    public static final ResourceLocation GLASSBLOWER_ID    = new ResourceLocation(Constants.MOD_ID, "glassblower");
+    public static final ResourceLocation DYER_ID           = new ResourceLocation(Constants.MOD_ID, "dyer");
+    public static final ResourceLocation MECHANIST_ID      = new ResourceLocation(Constants.MOD_ID, "mechanist");
+    public static final ResourceLocation FLETCHER_ID       = new ResourceLocation(Constants.MOD_ID, "fletcher");
+    public static final ResourceLocation CONCRETE_MIXER_ID = new ResourceLocation(Constants.MOD_ID, "concretemixer");
+    public static final ResourceLocation RABBIT_HERDER_ID  = new ResourceLocation(Constants.MOD_ID, "rabbitherder");
+    public static final ResourceLocation PLANTER_ID        = new ResourceLocation(Constants.MOD_ID, "planter");
+    public static final ResourceLocation BEEKEEPER_ID      = new ResourceLocation(Constants.MOD_ID, "beekeeper");
+
+    public static BipedModelType SETTLER;
+    public static BipedModelType CITIZEN;
+    public static BipedModelType NOBLE;
+    public static BipedModelType ARISTOCRAT;
+    public static BipedModelType BUILDER;
+    public static BipedModelType DELIVERYMAN;
+    public static BipedModelType MINER;
+    public static BipedModelType LUMBERJACK;
+    public static BipedModelType FARMER;
+    public static BipedModelType FISHERMAN;
+    public static BipedModelType UNDERTAKER;
+    public static BipedModelType ARCHER_GUARD;
+    public static BipedModelType KNIGHT_GUARD;
+    public static BipedModelType BAKER;
+    public static BipedModelType SHEEP_FARMER;
+    public static BipedModelType COW_FARMER;
+    public static BipedModelType PIG_FARMER;
+    public static BipedModelType CHICKEN_FARMER;
+    public static BipedModelType COMPOSTER;
+    public static BipedModelType SMELTER;
+    public static BipedModelType COOK;
+    public static BipedModelType STUDENT;
+    public static BipedModelType CRAFTER;
+    public static BipedModelType BLACKSMITH;
+    public static BipedModelType CHILD;
+    public static BipedModelType HEALER;
+    public static BipedModelType TEACHER;
+    public static BipedModelType GLASSBLOWER;
+    public static BipedModelType DYER;
+    public static BipedModelType MECHANIST;
+    public static BipedModelType FLETCHER;
+    public static BipedModelType CONCRETE_MIXER;
+    public static BipedModelType RABBIT_HERDER;
+    public static BipedModelType PLANTER;
+    public static BipedModelType BEEKEEPER;
+
+    private ModModelTypes()
+    {
+        throw new IllegalStateException("Tried to initialize: ModModelTypes but this is a Utility class.");
+    }
+}
```

Comment:
Again not the way this should be implemented.
You are exposing that BipedModelType which is completely not needed at all.

If this needs to be extendable, make a registry (any will do, even a `ConcurrentHashMap` and have everybody use that) That way you only need to expose the IDs as constants and not the underlying objects, reducing the chance for binary compatibility breaking bugs later on.

Reply:
Hmm, I never thought of that. Will make changes shortly.